Libplanet.Net/Transports/BoundPeerExtensions.cs

```diff
@@ -76,11 +77,13 @@ internal static async Task<string> ResolveNetMQAddressAsync(this BoundPeer peer)
             // FIXME Dns.GetHostAddressesAsync() seems to always return at least one ip or throw
             // `SocketException`. if we can be sure, also should fix it to always assume there
             // is at least one instead of throwing additional `TransportException`.
-            string ip = addresses.FirstOrDefault()?.ToString()
+            // And, for now IPs are restricted with IPv4, and have to be fixed later on.
```

Comment:
I think `XML` documentation would make this method more usable (that this method only checks whether `BoundPeer.EndPoint.Host` resolves to an IPv4 address).

Also looking at (although this is for `GetHostEntry()` instead of `GetHostAddressesAsync()`)

https://stackoverflow.com/questions/156585/can-dns-gethostentry-ever-return-an-iphostentry-with-an-empty-addresslist

seems like it is technically possible to get an empty list. Maybe we can also change the `FIXME` comment accordingly. 😗